### ISSUE TITLE
there is a problem...

### ISSUE THREAD

**Issue by iSlodinx** [MinisBett/ultimate-osu-server-switcher#9]
![Capture](https://user-images.githubusercontent.com/35035977/85045766-b72f6e00-b18f-11ea-98e7-376473525432.JPG)

i don't know whats the problem, i already downloaded .NET Framework and stuff but still nothing...

**Comment by MinisBett** [comment]
Please add the details of the error window to this issue.

**Comment by MinisBett** [comment]
Tracked down the issue to
```csharp
using (Stream stream = m_client.OpenRead(url))
return Image.FromStream(stream);
```

https://github.com/MinisBett/ultimate-osu-server-switcher/blob/master/UltimateOsuServerSwitcher/MainForm.cs#L301-L311

**Comment by MinisBett** [comment]
![image](https://user-images.githubusercontent.com/39670899/85255146-ff85af00-b461-11ea-8f3f-e75ba359b311.png)

**Comment by MinisBett** [comment]
Will be fixed in version 1.4

//closed
